packages/uix-host-react/src/hooks/useExtensions.ts

```diff
@@ -131,6 +136,17 @@ export function useExtensions<
     const newExtensions = [];
     const guests = host.getLoadedGuests(requires);
     for (const guest of guests) {
+      //proceed only if this guest supports provided extension point
+      if (
+        currentComponentExtensionPoint &&
+        guest.extensionPoints &&
+        !guest.extensionPoints.includes(
+          `aem/${currentComponentExtensionPoint}/1`
```

Comment:
Leaked abstraction: SDK must not be aware of AEM. Top-level product identifiers and versions cannot be hardcoded at this level.

Reply:
I forgot to remove hardcoding, will update code right away. I agree with SDK must not be aware of aem or any other user of library.